turbinia/turbiniactl.py

```diff
@@ -161,6 +161,11 @@ def main():
       required=False)
   parser_rawdisk.add_argument(
       '-n', '--name', help='Descriptive name of the evidence', required=False)
+  parser_rawdisk.add_argument(
```

Comment:
Hi @youjitwo !  Welcome to Turbinia and thanks/congrats on your first PR :)

I know this PR is still in draft mode, but I just wanted to make a high-level comment here that these flags are not specific to any piece of evidence, so we could probably add them to the base flags instead of each sub-command.  See `--request_id` as a related example.  Thanks!